tests/src/FunctionalJavascript/ContributionPayLaterTest.php

```diff
@@ -74,15 +75,19 @@ public function testReceiptParams() {
     $this->assertStringContainsString('Payment by Direct Credit to: ABC. Please quote invoice number and name.', $sent_email);
     $this->assertStringContainsString('Thank you for your contribution', $sent_email);
 
+    // "Hack" for versions before 5.54 - it doesn't pick up the setting we set earlier.
+    $mb = \Civi::settings()->get('mailing_backend');
```

Comment:
Do we need to wrap this in a version check then?

Reply:
makes sense